about.md

```diff
@@ -0,0 +1,58 @@
+# About the Project
+
+One critical aspect of an undergraduate STEM education is hands-on research. Undergraduate research experiences enhance what students learn in the classroom as well as increase a student's interest in pursuing STEM careers [Russell et al. 2007][1]. It can also lead to improved scientific reasoning and increased academic performance overall [Buffalari et al. 2020][2]. However, many students at underresourced institutions like community colleges, Historically Black Colleges and Universities (HBCUs), tribal colleges and universities, and Hispanic-serving institutions have limited access to research opportunities compared to their cohorts at larger four-year colleges and R1 institutions. These students are also more likely to belong to groups that are already under-represented in STEM disciplines, particularly genomics and data science [Canner et al. 2017][3],[GDSCN 2022][4]. 
+
+For this project, we're examining how the soil microbes vary based on soil characteristics and land use. We collect soil samples for both metagenomic and chemical analysis, and then use the data to look for patterns. This project offers students an opportunity to participate in a real research project (from sample collection to data analysis) as well as training in genomics methods, training in data science, and training in using cloud computing resources like AnVIL.
```

Comment:
Maybe we want to be more general here to keep it within NHGRI scope, and just say this project aims to be at the intersection of genomics, ds, cloud computing, and education. Then get into the details below?